apps/frontend/src/components/volunteer/signup/SignUpPage.tsx

```diff
@@ -0,0 +1,216 @@
+                  >
+                    {field.label}
+                  </Text>
+                  <Input
+                    variant="filled"
+                    height={group.height}
+                    placeholder={field.placeholder || 'example'}
+                    width="100%"
+                  />
+                </VStack>
+              ))}
+            </HStack>
+          ) : (
+            <VStack width={group.width} align="flex-start">
+              <Text
+                className="label"
+                fontSize="18px"
+                fontWeight={600}
+                alignSelf="flex-start"
+                marginBottom={0}
+                marginTop="30px"
+              >
+                {group.fields[0].label}
+              </Text>
+              <Input
+                variant="filled"
+                height={group.height}
+                placeholder={group.fields[0].placeholder || 'example'}
+                width="100%"
+              />
+            </VStack>
+          )}
+        </VStack>
+      ))}
+    </VStack>
+  );
+}
+
+function CheckboxFields() {
+  return (
+    <VStack marginTop={'20px'}>
+      {checkboxesMap.map((field, i) => (
+        <HStack key={i} width="100%" height="100%" marginTop={'20px'}>
+          <Text textDecoration="underline" fontSize="18px" fontWeight={600}>
+            {field.label}
+          </Text>
+          <Checkbox
+            sx={{
+              color: '#808080', // Grey color for the checkbox when not checked
+              '&.Mui-checked': {
+                color: '#808080', // Grey color for the checkbox when checked
+              },
+              '& .MuiSvgIcon-root': {
+                fontSize: 30,
+              },
+            }}
+          />
+        </HStack>
+      ))}
+    </VStack>
+  );
+}
+
+export default function SignUpPage() {
+  return (
+    <Box
+      className="outermost-box"
+      position="absolute"
+      top="10%"
+      left="10%"
+      display="flex"
+      alignItems="center"
+      justifyContent="center"
+      bg="#D9D9D9"
+      width="80%"
+      height="250%"
+      zIndex={'200'}
+    >
+      <Box
+        className="inner-box"
+        bg="#FFFDFD"
+        height="90%"
+        width="90%"
+        display="flex"
+        flexDirection="column"
+        alignItems="center"
+        justifyContent="space-evenly"
+        paddingTop={'30px'}
+      >
+        <Box
+          className="header-box"
+          height="5%"
+          width="90%"
+          borderBottom="2px solid #000000"
+          display="flex"
+          justifyContent="center"
+          alignItems="center"
+        >
+          <Text
+            fontFamily="Montserrat"
+            fontSize="30px"
+            fontWeight={600}
+            paddingTop={'50px'}
+            paddingBottom={'30px'}
+          >
+            Welcome, Volunteer!
+          </Text>
+        </Box>
+        <Box className="input-fields-main" width="90%" mt="20px">
```

Comment:
I think it would be great if we could add a close button and a submit button as well! I don't think it was the design but probably good to add. You don't have to implement the functionality for submit right now.